src/openvic-simulation/pop/Pop.cpp

```diff
@@ -145,11 +353,15 @@ bool PopManager::load_pop_type_file(
 	Strata const* strata = nullptr;
 	PopType::sprite_t sprite = 0;
 	Good::good_map_t life_needs, everyday_needs, luxury_needs;
+	PopType::income_type_t life_needs_income_type = NONE, everyday_needs_income_type = NONE, luxury_needs_income_type = NONE;
+	fixed_point_t life_needs_income_weight = 0, everyday_needs_income_weight = 0, luxury_needs_income_weight = 0;
 	PopType::rebel_units_t rebel_units;
+	Pop::pop_size_t max_size = 0, merge_max_size = 0;
```

Comment:
max value of pop_size_t as default?
using 0 as there is no max adds complexity.